### ISSUE TITLE
Xcode 13 error: 'shared' is unavailable in application extensions for iOS

### ISSUE THREAD

**Issue by byaruhaf** [huri000/SwiftEntryKit#359]
When compiling our application, I see an error like:

```
'shared' is unavailable in application extensions for iOS: Use view controller based solutions where appropriate instead.
```

<img width="1206" alt="Screenshot 2021-10-17 at 11 44 13" src="https://user-images.githubusercontent.com/4116539/137619368-486d8fda-2f7a-4a7a-afad-3ecf14647cbb.png">

Note that this was not happening in our app with Xcode 12, it started happening when I updated to Xcode 13.0. 
Apple requires the library itself to be annotated like so:

```swift
@available(iOSApplicationExtension, unavailable)
```

See: https://forums.swift.org/t/set-application-extension-api-only-on-a-spm-package/39333/11

This is related to a change in Xcode made by Apple deliberately and for which they provided annotation library authors have to add when declaring methods that would trigger that compilation error.

![image](https://user-images.githubusercontent.com/4116539/137619799-ef9e2325-298a-4972-b8f0-cbd40b36d40d.png)

Thank you in advance for your help :)!

**To Reproduce**
- Add SwiftEntryKit to project via SPM
- Build the project with Xcode 13

**Expected behavior**
No build error's

**Environment**
 - Xcode Version Version 13.0 (13A233)
 - Dependency Manager SPM
 - SwiftEntryKit Release 1.2.7

**Comment by huri000** [comment]
Hey! Thanks for reporting this issue. Unfortunately I’m a little short in bandwidth,  so it would be great if you could provide a sample app that I can use to reproduce this issue.

**Comment by byaruhaf** [comment]
Hey, you can reproduce this issue with Paul Hudson's [Unwrap]( https://github.com/twostraws/Unwrap)
 https://github.com/twostraws/Unwrap

**Comment by byaruhaf** [comment]
Hi, the issue is fixed with SwiftEntryKit Release 2.0.0, Thanks A lot 👍🏽.  Closing this issue now.
